cli_tools/gce_vm_image_import/cli/cli.go

```diff
@@ -0,0 +1,145 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
Why is this refactoring needed for this test? We already test image import wrapper directly in cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go where we call the binary itself.

Reply:
resolving (duplicate of previous comment)